Hud/Sprites/SpriteSheetMetadata.cs

```diff
@@ -2,19 +2,34 @@
 using OpenTK.Mathematics;
 
 namespace Hud.Sprites;
-
+/// <summary>
+/// Represents a sprite sheet metadata.
+/// </summary>
 [Serializable]
 public class SpriteSheetMetadata
 {
+    /// <summary>
+    /// Width of the sprite sheet.
+    /// </summary>
     [JsonPropertyName("width")]
     public int Width { get; set; }
 
+    /// <summary>
+    /// Height of the sprite sheet.
+    /// </summary>
     [JsonPropertyName("height")]
     public int Height { get; set; }
 
+    /// <summary>
+    /// Items in the sprite sheet.
+    /// </summary>
     [JsonPropertyName("items")]
     public List<Sprite> Items { get; set; } = null!;
 
+    /// <summary>
+    /// Gets the rectangles of the sprites in the sprite sheet. Rectangles describe the position of a sprite in the sprite sheet.
+    /// </summary>
+    /// <returns></returns>
```

Comment:
This is a rather complex return type, consider describing the elements of the tuple